src/DropdownReference/components/DropdownReferenceContainer.ts

```diff
@@ -0,0 +1,252 @@
+import { Component, createElement } from "react";
+import * as initializeReactFastclick from "react-fastclick";
+import { hot } from "react-hot-loader";
+
+import { AttributeType, DropdownReferenceProps, ReferenceOption, parseStyle, validateProps } from "../../SharedResources/utils/ContainerUtils";
+import { FetchDataOptions, Nanoflow, fetchData } from "../../SharedResources/utils/Data";
+import { DropdownReference } from "./DropdownReference";
+
+interface WrapperProps {
+    class: string;
+    mxObject: mendix.lib.MxObject;
+    mxform: mxui.lib.form._FormBase;
+    style: string;
+    readOnly: boolean;
+    friendlyId: string;
+}
+
+export interface ContainerProps extends WrapperProps, DropdownReferenceProps {
+    attribute: string;
+    entityPath: string;
+    entityConstraint: string;
+    searchAttribute: string;
+    source: "xpath" | "microflow" | "nanoflow";
+    sortOrder: "asc" | "desc";
+    sortAttributes: AttributeType[];
+    nanoflow: Nanoflow;
```

Comment:
```suggestion
    nanoflow: mx.Nanoflow;
```
https://github.com/FlockOfBirds/Mendix-client-typing/blob/master/index.d.ts#L339

Reply:
Done